spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/redis/RedisProperties.java

```diff
@@ -410,6 +410,13 @@ public Refresh getRefresh() {
 				 */
 				private boolean adaptive;
 
+				/**
+				 * Whether discovered nodes should be used as the source for the cluster
+				 * topology. When set to {@literal false}, only the initial seed nodes
+				 * will be used as sources for topology discovery.
+				 */
+				private Boolean dynamicSources = null;
```

Comment:
The `null` initialization here is unnecessary.  The AP will figure out the right default for a wrapper type.

Reply:
I thought so but I noticed it explicitly specified in places like [this](https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/mongo/MongoProperties.java#L59). I will remove the redundant initialization.